lib/telemetry_metrics_statsd.ex

```diff
@@ -300,8 +303,10 @@ defmodule TelemetryMetricsStatsd do
     config =
       options
       |> Enum.into(%{})
-      |> Map.put_new(:host, "localhost")
-      |> Map.update!(:host, &to_charlist/1)
+      |> Map.put_new(:host, {127, 0, 0, 1})
+      |> Map.update!(:host, fn host ->
+        if(String.valid?(host), do: to_charlist(host), else: host)
```

Comment:
```suggestion
        if(is_binary(host), do: to_charlist(host), else: host)
```

I'd just go with good ol' `is_binary/1` here - what do you think?

Reply:
Sure!